src/os/ui/featureedit.js

```diff
@@ -847,6 +862,21 @@ goog.exportProperty(
     'updatePreview',
     os.ui.FeatureEditCtrl.prototype.updatePreview);
 
+/**
+ * Updates the altitude units.
+ * @param {*} value
+ */
+os.ui.FeatureEditCtrl.prototype.updateAltUnits = function(value) {
```

Comment:
Is this still used? I don't see any reference to it in the PR. If not, please remove it.